effect/_intents.py

```diff
@@ -50,10 +46,23 @@ def parallel(effects):
     their results, in the same order as the input to this function.
 
     :param effects: Effects which should be performed in parallel.
+    :return: An Effect that results in a list of results, or which fails with
+        a :obj:`FirstError`.
     """
     return Effect(ParallelEffects(list(effects)))
 
 
+@attributes(['exception', 'index'])
```

Comment:
I'm unhappy with having two ways of representing exception. While I'm not sure that the `sys.exc_info` tuple is a *good* representation, I don't think diverging here is sensible.

It wouldn't be unreasonable to have an `exception` property, that deconstructs the tuple for convenience, though.